Src/Viewables/ContentInsuranceCard.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ContentInsuranceCard.swift
+//  test
+//
+//  Created by Pavel Barros Quintanilla on 2020-03-04.
+//
+
+import Foundation
+import Flow
+import Form
+import Presentation
+import UIKit
+
+enum ContentAction {
+    case toContentInsurance, toTravelInsurance
+}
+
+struct ContentInsuranceCard {
+    let iconImage: ImageAsset
+    let title: String
+    let description: String
+    let type: ContentAction
+    let presentingViewController: UIViewController
+}
+
+extension ContentInsuranceCard: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Future<Void>) {
```

Comment:
Make it into a signal instead, and then the consumer will have the responsibility for showing the subscreen.